tests/test_builders/test_build_html_5_output.py

```diff
@@ -1,17 +1,20 @@
 """Test the HTML builder and check output against XPath."""
 
 import re
+from collections.abc import Callable, Iterable
+from typing import Literal
+from xml.etree.ElementTree import Element
```

Comment:
Put them in a TYPE_CHECKING block instead.

Reply:
for reasons that are not obvious to me, ruff complains if these are moved into a type checking block

> Move import {IMPORT} out of type-checking block. Import is used for more than type hinting. [Ruff [TCH004](https://docs.astral.sh/ruff/rules/runtime-import-in-type-checking-block)]

Reply:
It's because `from future import __annotations__` is missing.